### ISSUE TITLE
Multiple dependencies do not work in NimScript format

### ISSUE THREAD

**Issue by endragor** [nim-lang/nimble#262]
Tried this statement: `requires "uuids >= 0.1.0, isaac >= 0.1.0"` 
It produced this error:

```
Reading as ini file failed with:
    Invalid section: .
Evaluating as NimScript file failed with:
  Unable to parse dependency version range: Unexpected char in version range: ,.
```

Putting the same string in .ini format works fine.
Nimble v0.7.10.

**Comment by yglukhov** [comment]
IMHO I would not nosider that an issue. Though maybe doc should be more clear about it.

**Comment by dom96** [comment]
I agree with @yglukhov. But I do think that the docs are fairly adequate, ideas on how to improve them are welcome though.

**Comment by yglukhov** [comment]
Just remove any mentions about the ini format and samples that show possibility to use comma in `Requires:`, and that should be it =)

**Comment by endragor** [comment]
The docs do not mention at all that multiple `requires` is allowed, unless I missed it.

I would love if the "requires" word is removed completely. At the moment specifying many dependencies gets a bit too wordy:
```
requires "uuids >= 0.1.0 & < 0.2.0"
requires "isaac >= 0.1.0 & < 0.2.0"
```
Something like this would be nice:
```
requires:
  uuids = ^0.1.0
  isaac = ^0.1.0
```

**Comment by dom96** [comment]
You can do this:

```nim
requires "uuids >= 0.1.0 & < 0.2.0", "isaac >= 0.1.0 & < 0.2.0"
```
